<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>861</x>
     <y>1246</y>
     <width>296</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>加载</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStartMaster">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开启master</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_slaves">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_M3_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>已连接的slave信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_M3_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>当前M区数据</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_mData">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="HLed" name="alarmPump" native="true">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="HLed" name="alarmGen" native="true">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>40</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="HLed" name="alarmTurbine" native="true">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>水泵液高报警</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>汽轮机转速报警</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>发电机温度报警</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelAlarm">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消所有报警</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HLed</class>
   <extends>QWidget</extends>
   <header>hled.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
